<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>618</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(69, 196, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="textLog">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>741</width>
      <height>191</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>741</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="basic">
     <attribute name="title">
      <string>Basic</string>
     </attribute>
     <widget class="QLabel" name="labelCurrentIP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>301</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Your current global IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelConfig">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Config is: undefined</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Actions</string>
      </property>
      <widget class="QPushButton" name="buttonCheckConfig">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>91</width>
         <height>33</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>Check Config</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonRefreshIP">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>91</width>
         <height>33</height>
        </rect>
       </property>
       <property name="text">
        <string>Refresh IP</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="config">
     <attribute name="title">
      <string>Config</string>
     </attribute>
     <widget class="QCheckBox" name="telegramsupportCheckbox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram Notifications</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSaveConfig">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>270</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Config</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dynu API Key</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Domain ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram API Key</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Chat ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dynuapikeyedit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="domainidedit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="chatidedit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="telegramapikeyedit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="domainnameedit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>domain.dynu.net</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Domainname</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select your language:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="languageComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>94</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
